packages/@lwc/engine-core/src/framework/base-lightning-element.ts

```diff
@@ -495,65 +474,84 @@ LightningElement.prototype = {
 
 const queryAndChildGetterDescriptors: PropertyDescriptorMap = create(null);
 
-const childGetters: Array<
-    [
-        keyof HTMLElement,
-        keyof Pick<
-            Renderer,
-            | 'getChildren'
-            | 'getChildNodes'
-            | 'getFirstChild'
-            | 'getFirstElementChild'
-            | 'getLastChild'
-            | 'getLastElementChild'
-        >
-    ]
-> = [
-    ['children', 'getChildren'],
-    ['childNodes', 'getChildNodes'],
-    ['firstChild', 'getFirstChild'],
-    ['firstElementChild', 'getFirstElementChild'],
-    ['lastChild', 'getLastChild'],
-    ['lastElementChild', 'getLastElementChild'],
+// The reason we don't just call `import * as renderer from '../renderer'` here is that the bundle size
+// is smaller if we reference each function individually. Otherwise Rollup will create one big frozen
+// object representing the renderer, with a lot of methods we don't actually need.
+const childGetters = <const>[
```

Comment:
TIL that this is a valid typescript syntax. Nitpicking on this, a more widely used syntax to express this would be:

```ts
const childGetters = [
    'children',
    'childNodes',
    'firstChild',
    'firstElementChild',
    'lastChild',
    'lastElementChild',
] as const; 
```

Reply:
Thanks, I'll use that. I'm really not familiar with TypeScript. 😆